pkg/plugins/github/pullrequest.go

```diff
@@ -247,6 +255,28 @@ func (p *PullRequest) updatePullRequest() error {
 	return nil
 }
 
+// EnablePullRequestAutoMerge updates an existing pullrequest with the flag automerge
+func (p *PullRequest) EnablePullRequestAutoMerge() error {
+
+	var mutation struct {
+		EnablePullRequestAutoMerge struct {
+			PullRequest PullRequestApi
+		} `graphql:"enablePullRequestAutoMerge(input: $input)"`
+	}
```

Comment:
The code looks good. Eventhough the "pullrequest" object is not ready for (yet) for dependency injection to allow decoupling from githubv4 + mocking for unit test, do you mind defining this type as package-private in the same way as it is done in the githib plugin: https://github.com/updatecli/updatecli/blob/main/pkg/plugins/github/release.go#L46-L66 
?

It would allow factorization a bit easier in the future (and the tests, either integ. or unit could access the type)

Reply:
I am not sure to understand what you are asking for. Are you asking me to refactor the code in pullrequest.go file?

Reply:
I'm only asking you to declare the type of the variable `mutation` as a package type (but a private one).

So that mutation declaration would be something like:

```
var mutation mutationQuery
```

However if you want to jump on the "unit test" wagon, you can refactor the pullrequest.go ;)

Reply:
As requested, I created the package struct `mutationEnablePullRequestAutoMerge`. I won't have the time to refactor the package to implement UT so I propose to move forward with the current state.